ggplot2_matplotlib_cheatsheet.Rmd

```diff
@@ -0,0 +1,13 @@
+# Data Visualization with GGPlot2 (R) and MatPlotLib (Python) Cheat Sheet
+
+Motivation
```

Comment:
The second line should be blank, followed by the author name(s).

Reply:
I have added the author name to the 3rd line of code